source/bson-corpus/tests/decimal128-5.json

```diff
@@ -151,6 +151,12 @@
             "degenerate_extjson": "{\"d\" : {\"$numberDecimal\" : \"0E-6177\"}}",
             "canonical_extjson": "{\"d\" : {\"$numberDecimal\" : \"0E-6176\"}}"
         },
+        {
+            "description": "clamped zeros with a large negative exponent",
+            "canonical_bson": "180000001364000000000000000000000000000000000000",
+            "degenerate_extjson": "{\"d\" : {\"$numberDecimal\" : \"0E-999999999999\"}}",
+            "canonical_extjson": "{\"d\" : {\"$numberDecimal\" : \"0E-6176\"}}"
+        },
```

Comment:
Was the reason for adding these tests to `decimal128-5.json` because they copy the preceding test case and just modify `degenerate_extjson`?

The tests in this file are taken from `dqBase.decTest` in `decTest.zip` from https://speleotrove.com/decimal/dectest.html (as referenced in the [Decimal128 spec](https://github.com/mongodb/specifications/blob/master/source/bson-decimal128/decimal128.rst#tests)).

```
-- clamped zeros [see also clamp.decTest]
dqbas1075 toSci   0e+10000  ->  0E+6111 Clamped
dqbas1076 toSci   0e-10000  ->  0E-6176  Clamped
dqbas1077 toSci  -0e+10000  -> -0E+6111 Clamped
dqbas1078 toSci  -0e-10000  -> -0E-6176  Clamped
```

I think there's value in keeping these "ported" files as-is and adding these tests either a new file or the end of an existing one. `decimal128-1.json` and `decimal128-6.json` look to be organic tests not ported from the aforementioned test suite.

That said, I won't protest if everyone else prefers keeping them here.

Reply:
There wasn't a clear pattern for where to add new `decimal128` tests, so I just added them near existing similar tests. However, I can definitely add them to either `decimal128-1.json` or `decimal128-6.json` instead.